requirements/edx/base.txt

```diff
@@ -168,7 +168,7 @@ numpy==1.18.2             # via chem, openedx-calc, scipy
 git+https://github.com/joestump/python-oauth2.git@b94f69b1ad195513547924e380d9265133e995fa#egg=oauth2  # via -r requirements/edx/github.in
 oauthlib==3.0.1           # via -c requirements/edx/../constraints.txt, -r requirements/edx/base.in, django-oauth-toolkit, lti-consumer-xblock, requests-oauthlib, social-auth-core
 openedx-calc==1.0.7       # via -r requirements/edx/base.in
-git+https://github.com/edx/edx-ora2.git@2.6.19#egg=ora2==2.6.19  # via -r requirements/edx/github.in
+git+https://github.com/edx/edx-ora2.git@2.6.19#egg=ora2==2.6.20  # via -r requirements/edx/github.in
```

Comment:
Actually, I was wrong to approve this - you have to make sure that the patch version gets bumped on both sides of the `#`.  No biggie, just watch out for it in the future.